### PR TITLE
Implement progress spinner after wizard

### PR DESCRIPTION
**Pull request by TheOneRing**
image

Fixes: #10751

### CODE REVIEW

**Comment by TheOneRing** [comment]
The design is questionable and the spinner makes it more obvious that the ui might freeze..

File: src/gui/accountsettings.ui [hunk 1]
```diff
@@ -50,22 +50,80 @@
...
+           <width>20</width>
+           <height>40</height>
+          </size>
+         </property>
+        </spacer>
+       </item>
+       <item>
+        <widget class="QLabel" name="label">
+         <property name="text">
+          <string>Getting ready</string>
```
Comment (fmoc) [inline src/gui/accountsettings.ui:75]: "Preparing account..." might be a more elegant wording. "Readying account" would work, too. We should avoid the use of "Get".

File: src/gui/owncloudgui.cpp [hunk 2]
```diff
@@ -897,19 +898,21 @@ void ownCloudGui::runNewAccountWizard()
 
                                     folderMan->setSyncEnabled(true);
                                     folderMan->scheduleAllFolders();
+                                    accountStatePtr->setSettingUp(false);
```
Comment (fmoc) [inline src/gui/owncloudgui.cpp:901]: Just a question in general, would it be more elegant to use a setter like this or rather emit a signal on the account state pointer? I saw we need to track the state anyway in a local variable.

**Review by fmoc — CHANGES_REQUESTED** [review]

File: src/gui/owncloudgui.cpp (hunk 2 above)
Comment (TheOneRing) [reply src/gui/owncloudgui.cpp:901]: Emitting signals for other objects feels "wrong" additionally the setter has the benefit that we can query the state and not rely only on signals.

**Review by fmoc — APPROVED** [review]

**Pull request merged**